src/components/Nav/Nav.js

```diff
@@ -1,5 +1,62 @@
 import React from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import Search from './components/Search/Search';
+import * as S from './Nav.style';
 
 export default function Nav() {
-  return <div>Nav</div>;
+  const userToken = localStorage.getItem('UserToken');
+  const navigate = useNavigate();
+
+  const toPosting = () => {
+    if (userToken === null) {
```

Comment:
```suggestion
    if (!userToken) {
```
! 연산자를 붙이면 코드가 줄어듭니다